ledger/ledger-runner-common/src/test/scala/com/daml/ledger/runner/common/PureConfigReaderWriterSpec.scala

```diff
@@ -750,15 +750,8 @@ class PureConfigReaderWriterSpec
 
   val validIndexServiceConfigValue =
     """
-      |  acs-contract-fetching-parallelism = 2
```

Comment:
We are no testing the nested stream configs here

Reply:
Ok, added tests